src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -72,8 +72,13 @@ public void setupRoutes(Router router) {
                 this.handleRewriteMetadata(ctx);
             }
         }, Role.CLIENTKEY_ISSUER));
+
         router.get("/api/client/list").handler(
                 auth.handle(this::handleClientList, Role.CLIENTKEY_ISSUER));
+
+        router.get("/api/client/listBySite/:siteId").handler(
```

Comment:
following the pattern in [SiteService.java](https://github.com/IABTechLab/uid2-admin/blob/86edd40783a9b3e5930d97917fce710619bbea1c/src/main/java/com/uid2/admin/vertx/service/SiteService.java#L64), perhaps just `/api/client/:siteId` would be better?

Reply:
I think that route may become misleading. I would think it would possibly more look like that endpoint would provide a single key based on the id?

Reply:
what about just `/api/client/list/:siteId`? That pattern is also used in `SharingService.java`

Reply:
Sounds good. Have made that change.